rules/cross-approvals/rule.md

```diff
@@ -27,14 +28,17 @@ In small companies, a single key stakeholder often approves everything, ensuring
 
 ## What is an approval?
 
-Before we dive into the benefits of scaling approvals. It's worth establishing a baseline for what an approval is. Generally an approval is anytime you need someone else to check a piece of work you have done before it can be marked as done.
+Generally an approval is anytime you need someone else to check a piece of your work before it can be marked as done.
 
-## Benefits of scaling approvals
+Approvals can either be:
 
-✅ Significant time savings for the person causing the bottlenecks (aka the "original approver")
+* Ad hoc - these are best done following the 'checked by' rule: [Do you use 'Checked by xxx'?](https://www.ssw.com.au/rules/checked-by-xxx/)
+* Recurring - these cause "approval hell" - you should use **approval scaling**
 
-✅ Fewer bottlenecks for those seeking approval
+## Benefits of scaling approvals
 
+✅ Time savings for the person causing the bottlenecks (aka the "original approver")
+✅ Fewer bottlenecks for those seeking approval
 ✅ Consistent and high-quality approvals
```

Comment:
```suggestion
✅ Time savings for the person causing the bottlenecks (aka the "original approver")

✅ Fewer bottlenecks for those seeking approval

✅ Consistent and high-quality approvals
```

Reply:
I think you need to leave empty lines otherwise they will all be on the same line. Like you did in the third part "Classify - Determine the importance of the task"